.rubocop_todo.yml

```diff
@@ -1,113 +1,140 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2016-12-11 13:27:14 +0100 using RuboCop version 0.46.0.
+# on 2019-06-06 14:12:57 -0700 using RuboCop version 0.54.0.
```

Comment:
Please rebase your branch with master before. We currently use the latest version of rubocop.